cmd/makedocs/main.go

```diff
@@ -160,8 +152,41 @@ func parseNote(lines []string, index *int, c *checker) error {
 	}
 	*index++
 	for *index < len(lines) && len(lines[*index]) > 0 {
-		c.Note += lines[*index] + "\n"
+		c.Note += lines[*index]
 		*index++
 	}
 	return nil
 }
+
+func getPkgs(path string, suffix string) map[string]*ast.Package {
+	filter := func(inf os.FileInfo) bool {
+		return strings.HasSuffix(inf.Name(), suffix)
+	}
+
+	pkgs, err := parser.ParseDir(&token.FileSet{}, path,
+		filter, parser.ParseComments)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return pkgs
+}
+
+func parseComments(file *ast.File, cd *checkerDoc) {
+	for _, comment := range file.Comments {
+		if strings.HasPrefix(comment.Text(), "!") {
+			parseComment(comment.Text(), cd)
+			break
+		}
+	}
+}
+
+func parseFiles(path string) []checkerDoc {
```

Comment:
unused, that's why you've a failing build

Reply:
TY.